massa-execution-worker/src/context.rs

```diff
@@ -9,12 +9,15 @@
 
```

Comment:
general remark: we need to prevent deferred messages to be sent with 0 gas to avoid flooding. There should be a minimal gas (eg. VM launch gas). Check if and how we did that for async messages as well

Reply:
* https://github.com/massalabs/massa/blob/df245a8bfa615fedd2e34489fd6777206027c478/massa-execution-worker/src/interface_impl.rs#L1475